src/test/cli.ts

```diff
@@ -29,6 +29,7 @@ const defaultReporter = process.env.CI ? 'dot' : 'list';
 const builtinReporters = ['list', 'line', 'dot', 'json', 'junit', 'null'];
 const tsConfig = 'playwright.config.ts';
 const jsConfig = 'playwright.config.js';
+const mjsConfig = 'playwright.config.mjs';
```

Comment:
Please also update documentation, for example https://playwright.dev/docs/next/intro#create-a-configuration-file.

Reply:
I think we don't need to call out `.mjs` in the docs. It works for consistency but it's really just noise in the docs.